k8s/pkg/apis/longhorn/v1beta2/node.go

```diff
@@ -32,6 +32,19 @@ const (
 	DiskConditionReasonDiskNotReady          = "DiskNotReady"
 )
 
+const (
+	SchedulerErrorInsufficientStorage          = "insufficient storage"
+	SchedulerErrorDiskNotFound                 = "disk not found"
+	SchedulerErrorNoDisAvailable               = "no disk available"
+	SchedulerErrorSettingIOError               = "setting io error"
+	SchedulerErrorTagsNotFullfilled            = "tags not fullfilled"
+	SchedulerErrorNodeNotFound                 = "node not found"
+	SchedulerErrorNoNodeAvailable              = "none of the nodes is available"
+	SchedulerErrorEngineImageNotReady          = "none of the node candidates contains a ready engine image"
+	SchedulerErrorHardAntiAffinityNotSatisfied = "hard anti-affinity cannot be satisfied"
```

Comment:
Change to alias with the spec property -> `SchedulerErrorHardNodeAffinityNotSatisfied`, the same as the description needed to adjust.